### PR TITLE
rm: boards: Add notes to help with secure boot

### PR DESCRIPTION
**Pull request by angolini**
Signed-off-by: Daiane Angolini <daiane.angolini@foundries.io>

### CODE REVIEW

**Comment by doanac** [comment]
Docs for 4409656f883dc051c2ed4d23a249826da8d50ab4 are browesable at: https://ci.foundries.io/projects/fio-docs/builds/1104/docs/artifacts/html/index.html

File: source/reference-manual/boards/secure-boot-note.rst [hunk 1]
```diff
@@ -0,0 +1,7 @@
+.. note::
+    The instructions on this section also applies to those boards with secure
```
Comment (Tim-Anderson) [inline source/reference-manual/boards/secure-boot-note.rst:2]: instructions in

File: source/reference-manual/boards/secure-boot-pre-flash-note.rst [hunk 2]
```diff
@@ -0,0 +1,5 @@
+.. note::
+
+     For instructions on how to sign the required images before flashing then
```
Comment (Tim-Anderson) [inline source/reference-manual/boards/secure-boot-pre-flash-note.rst:3]: change then to them

**Review by Tim-Anderson — CHANGES_REQUESTED** [review]
simple edits and then should be good to go

**Review by vanmaegima — APPROVED** [review]
please fix Tim's comments, then lgtm

**Comment by doanac** [comment]
Docs for cd9dfb077d98dda768c200f8e04cc55401978861 are browesable at: https://ci.foundries.io/projects/fio-docs/builds/1107/docs/artifacts/html/index.html

**Review by Tim-Anderson — APPROVED** [review]
LGTM

**Review by munoz0raul — APPROVED** [review]
LGTM

**Review by MrCry0 — APPROVED** [review]
LGTM

**Pull request merged**